### ISSUE TITLE
Unable to request access with different environment when user is part of both environments

### ISSUE THREAD

**Issue by mvidhu** [data-dot-all/dataall#510]
### Describe the bug

User is part of env A and env B. User is requesting access to a dataset created in env A by selecting env B so they can create dashboard in env B. However submit button is not visible in Shares tab to complete the share request after selecting the tables. 

### How to Reproduce

```
*P.S. Please do not attach files as it's considered a security risk. Add code snippets directly in the message body as much as possible.*
```
In this scenerio user is part of both Team A and Team B.
Link env A to data.all using team A.
Link env B to data.all using team B.
Import dataset A from env A to data.all
Create share request to dataset A using env as env B.
 

### Expected behavior

We expect to submit share request on behalf of team B , env B to env A.

### Screenshots

![Screenshot 2023-06-09 at 14 02 15](https://github.com/awslabs/aws-dataall/assets/87707075/43e99beb-d4b3-4c11-be1a-7f4511fd0e79)

### OS

Mac

### Python version

3.11

### AWS data.all version

1.5.4

**Comment by dlpzx** [comment]
Hi @mvidhu,

I see that you upgraded to v1.5.4 🚀 

The issue is that the requesters did not submit the request. From what I see in the request, the request itself is still in `DRAFT`, they need to click on `Submit`

Let me know if this solves the issue or if you need more details

**Comment by mvidhu** [comment]
Requestor is not seeing a submit button , that is the issue here

**Comment by dlpzx** [comment]
This issue was solved by #793
